Engines/Wine/QuickScript/Uplay Script/script.js

```diff
@@ -53,12 +54,14 @@ UplayScript.prototype.go = function () {
         .wizard(setupWizard)
         .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
         .luna();
+        .corefonts();
```

Comment:
Same comment as for steam

Reply:
![image](https://user-images.githubusercontent.com/11302521/59020549-d327d200-880f-11e9-81e1-f3e46c614a04.png)
ehhhhhh...

Reply:
Ah QA for syntax, mb

Should work assuming that it works this way for luna verb

Reply:
Verify it.

Reply:
you ware right -.-, fixed